config/api-postman.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+return [
+
+    /**
+     * Structured
+     *
+     * If you want folders to be generated based on namespace.
+     */
+
+    'structured' => false,
+
+    /**
+     * Base URL
+     *
+     * The base URL for all of your endpoints.
+     */
+
+    'base_url' => 'https://api.example.com/',
```

Comment:
Let's use the `APP_URL` env variable here instead of making them define it twice. 😉